freeipa/src/main/java/com/sequenceiq/freeipa/converter/instance/template/InstanceTemplateRequestToTemplateConverter.java

```diff
@@ -73,10 +73,15 @@ public Template convert(InstanceTemplateRequest source, CloudPlatform cloudPlatf
     }
 
     private void setVolumesProperty(Set<VolumeRequest> attachedVolumes, Template template, CloudPlatform cloudPlatform) {
+        String type = "HDD";
+        if (CloudPlatform.GCP == cloudPlatform) {
```

Comment:
maybe this should go the interface layer as a method like getFreeIPADefaultDiskType() and then it will be implemented in each cloudprovider modul ?

Reply:
@doktoric How about implementing like https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/java/com/sequenceiq/cloudbreak/service/stack/DefaultRootVolumeSizeProvider.java? That way I can contain the change within FreeIPA.

I could not find how gp2 and Standard SSD LRS are chosen for AWS and Azure. They are certainly not honoring the HDD value here.

Reply:
Thanks, @doktoric, followed that approach.